tests/gunicorn/tox.ini

```diff
@@ -20,11 +20,11 @@ whitelist_externals =
 setenv =
   PYTHONPATH=../../src
   ENABLE_INSTRUMENTATION=False
-  HT_REPORTING_TRACE_REPORTER_TYPE=zipkin
+  HT_TRACES_EXPORTER=zipkin
```

Comment:
We changed this env var to HT_REPORTING_TRACE_REPORTER_TYPE

Reply:
Fixed